### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by LiluUncharted**
Homework 3

### CODE REVIEW

File: .gitignore
Comment (IvanCeban) [inline .gitignore:1]: The "local" folder and it's content should  be added in the gitignore file. Same as the teamp.txt file.

Comment (IvanCeban) [reply .gitignore:1]: Start getting used to writing the commit messages in English. Not critical at the moment, but good to start doing that for future

**Review by IvanCeban — COMMENTED** [review]
local/local.txt
ignores only local.txt file inside local folder. 
In order to ignore everything inside the local folder, you need to write just "local/"

**Comment by LiluUncharted** [comment]
done

  
>Понедельник, 14 августа 2023, 15:15 +03:00 от Ivan Ceban ***@***.***>:
> 
> 
>@IvanCeban commented on this pull request.
>local/local.txt
>ignores only local.txt file inside local folder.
>In order to ignore everything inside the local folder, you need to write just "local/"
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you authored the thread. Message ID:  <LiluUncharted/Firstproject/pull/1/review/1576728518 @ github . com> 
 
 
С уважением,
Valeria Lazari
***@***.***

**Pull request closed without merging**